#ifndef PROTOINC_H_
#define PROTOINC_H_

#include "User.pb.h"
#include "ErrorRet.pb.h"
#include "ProtoNotify.pb.h"
#include "ProtoActivity.pb.h"
#include "ProtoBuilding.pb.h"
#include "ProtoPush.pb.h"
#include "DataCommon.pb.h"
#include "ProtoGM.pb.h"
#include "ProtoOrder.pb.h"
#include "ProtoShop.pb.h"
#include "ProtoAdvertise.pb.h"
#include "ProtoFriend.pb.h"
#include "ProtoTask.pb.h"
#include "ProtoAlliance.pb.h"
#include "ProtoNPCSeller.pb.h"
#include "ProtoReward.pb.h"
#include "ProtoVIP.pb.h"
#include "ProtoAssistor.pb.h"
#include "ProtoMission.pb.h"
#include "ProtoNPCCustomer.pb.h"
#include "ProtoRandomBox.pb.h"
#include "ProtoMailDog.pb.h"
#include "ProtoRotaryTable.pb.h"
#include "ProtoFriendlyTree.pb.h"
#include "ProtoActivityTencent.pb.h"
#include "ProtoAccessAd.pb.h"
#include "ProtoTheme.pb.h"
#include "ProtoActivity.pb.h"
#include "ProtoKeeper.pb.h"
#include "ProtoCard.pb.h"
#include "ProtoFriendWorker.pb.h"
#include "ProtoPet.pb.h"
#endif /* PROTOINC_H_ */
